lib/rouge/lexers/perl.rb

```diff
@@ -178,6 +178,26 @@ def self.detect?(text)
         rule /;/, Punctuation, :pop!
       end
 
+      state :sq do
+        rule /\\[']/, Str::Single
+        rule /[^\\']+/, Str::Single
+        rule /'/, Punctuation, :pop!
+      end
+
+      state :dq do
+        mixin :string_intp
+        rule /\\[\\tnr"]/, Str::Double
+        rule /[^\\"]+?/, Str::Double
+        rule /"/, Punctuation, :pop!
+      end
+
+      state :bq do
+        mixin :string_intp
+        rule /\\[\\tnr`]/, Str::Backtick
```

Comment:
Did you mean to catch `whitespace` here? Wouldn't ```[\s`]``` be more readable and apt..?

Reply:
That line is for escaped characters. I don't think that line does catch whitespace.

Reply:
okay.. just confirming..
*Conversation has been resolved*

Reply:
You can use `Str::Escape` here though since it's already identified.

Reply:
True. Updated, thanks :)